app/views/about.html

```diff
@@ -0,0 +1,49 @@
+<div class="page-content">
+  <div class="row">
+    <!-- include the sidebar -->
+    <div ng-include="'views/sidebar.html'"></div>
+    <div class="col-md-10">
+      <div class="content-box-large">
+        
+        <div class="panel-heading">
+          <div class="panel-title">
+            <h2><i class="glyphicon glyphicon-info-sign"></i>&nbsp;&nbsp;About the OpenHIM</h2>
+            <p class="text-muted">
+              Details about the application.
+            </p>
+          </div>
+        </div>
+
+
+        <div class="panel-body">
+          <div>
+            <div class="version version-{{ aboutInfo.compatibility | lowercase }}" ng-if="aboutInfo && aboutInfo.compatibility">
```

Comment:
with the Boolean changes for Compatibility earlier, this will be better with ng-class to determine the class to use.